sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 # Release History
+## 5.0.2 (2021-01-14)
+### Dependency updates
+- Update dependency version, `azure-core` to 1.12.0 and `azure-core-http-netty` to 1.7.1.
+
+## 5.1.0-beta.3 (2020-11-19)
```

Comment:
I don't think we need to include the beta changes, right?

Reply:
Agreed, adding newer version beta release CHANGELOG entries don't make sense as for this release this code doesn't exist.

Reply:
will update it.